src/TraceEvent/Utilities/command.cs

```diff
@@ -419,19 +420,27 @@ public Command(string commandLine, CommandOptions options)
             process = new Process();
             process.StartInfo = startInfo;
             output = new StringBuilder();
+
+// The functions needed to implement elevation are not supported in netstandard1.6
+// The command may just work anyway if TraceEvent's process is already elevated
+#if ! NETSTANDARD1_6
```

Comment:
The library contains an 'isElevated' API so in the common case if you ask for elevation you need it.  

Thus this API is 'mostly broken'.  I would prefer in those cases that on NETSTANDARD1_6 we throw a clean NotImplementedException (indicating NetStandard1.6) to be unambiguous.